proximity_node:
  ros__parameters:
    pir_pin: 4
    pir_threshold: 0.5
    service_name: "motion"